pkg/sink/webhook.go

```diff
@@ -70,7 +70,7 @@ func NewWebhook(log logrus.FieldLogger, c config.Webhook, reporter AnalyticsRepo
 }
 
 // SendEvent sends event notification to Webhook url
-func (w *Webhook) SendEvent(ctx context.Context, event events.Event) (err error) {
+func (w *Webhook) SendEvent(ctx context.Context, event events.Event, _ []string) (err error) {
```

Comment:
I have a special comment here, maybe we can improve in the future. It seems not all the bot types requires event sources. To satisfy Single Responsibility Principle, instead of trying to satisfy Bot interface methods, we can come up with 2 intefaces as follows
```go
type Bot interface {
	Start(ctx context.Context) error
	IntegrationName() config.CommPlatformIntegration
	controller.
}

type SourcableBot interface {
	Bot
	SendEvent(ctx context.Context, event events.Event, sources []string)
}

type OutGoingBot interface {
	Bot
	SendEvent(ctx context.Context, event events.Event)
}
```

Reply:
Would have to dig deeper. But, ... yup... this is also legacy code and can be improved in the future.

Reply:
As I pointed in https://github.com/kubeshop/botkube/pull/691#discussion_r950047949, Elasticsearch should also support bindings, same with outgoing webhook 🙂 So that would mean this comment is no longer applicable.

Reply:
Elasticsearch and outgoing webhook source bindings now added.